### ISSUE TITLE
Migration to landscape2

### ISSUE THREAD

**Issue by tegioz** [ossf/ossf-landscape#390]
Hi! 👋

We've been working on a new version of the landscape generating software, [landscape2](https://github.com/cncf/landscape2). This new version relies on the same `landscape.yml` file as the main data source, so nothing changes regarding adding or updating entries in the landscape.

The [CNCF landscape](https://landscape.cncf.io/) has already been successfully migrated to landscape2, and we'd like to start migrating more landscapes 🙂

We are already building the OpenSSF landscape using landscape2, you can check it out at <https://openssf.landscape2.io/>. It'd be great if you could give it a try when you have a chance and, whenever you are ready, we can proceed with the migration.

The process is very simple and would involve the following steps:

- [ ] Add/update some DNS entries [1]
- [x] Stop background automated processes that write to this repository (i.e. CNCF bot)
- [ ] Unlink this repository from Netlify
- [ ] Update README file and clean up some files no longer needed in this repository
- [ ] Setup the [landscape2-validate-action](https://github.com/cncf/landscape2-validate-action) to validate the `landscape.yml` file on PRs (optional)

We can take care of the last 4 points, you'd only need to update the DNS entries.

Landscape2 supports embedding a view with items in a category/subcategory in another website, but it works slightly different than landscape1. If parts of this landscape are embedded using this mechanism on other websites, we'd appreciate if you could let them know about [how it works now](https://github.com/cncf/landscape2#embeddable-views).

Please ping me if you have any question 😉

[1] We’d need to know the hostnames you’d like to use to prepare our end to listen on them before switching the DNS (i.e. landscape.openssf.org). Once we are ready we'll provide you with the destination for the DNS entries so that you can update them.

**Comment by hythloda** [comment]
Can we test it out with landscape2@openssf.org first before proceeding?

**Comment by hythloda** [comment]
We are having issues with v2 and the at risk flag. We would like to monitor the v2 before trusting it completely as well. It looks like the landscape2 is pulling the correct at risk flagged member but not the new member it should.

**Comment by tegioz** [comment]
Could you share some more details about the problem? Not sure what you mean by the "at risk flag", but we'd be happy to look into it if there's any issue 🙂

**Comment by tegioz** [comment]
I see. But I'm afraid that's not related to landscape v2. V2 feeds from the same `landscape.yml` file and at the moment it's built daily. At build time, we'll use that data file as it is at that moment. But if there is something that's automatically adding or removing some entries from the data file they may not show up properly.

If you need us to build it manually at any time please let us know. <https://openssf.landscape2.io> it's being [built daily](https://github.com/cncf/landscape2-sites/actions/workflows/openssf-build-and-deploy.yml) and should be up to date (within that time frame).

**Comment by hythloda** [comment]
Our current code for the landscape:
<iframe src="https://landscape.openssf.org/pages/members/" frameBorder="0" id="landscape" scrolling="no" style="width: 1px; min-width: 100%"></iframe><script src="https://landscape.openssf.org/iframeResizer.js"></script>

**Comment by bbpursell1** [comment]
Currently, the only thing that the OpenSSF landscape is meaningfully used for is the display here: https://openssf.org/about/members/

That uses an iframe which displays this view: https://openssf.org/about/members/

Is there a similar view presented in landscape2?  I don't see anything displayed at https://openssf.landscape2.io/pages/members.

**Comment by tegioz** [comment]
Yes! It works slightly different but it should be easy to migrate. Please see:

<https://openssf.landscape2.io/embed-setup>
<https://github.com/cncf/landscape2/blob/main/README.md#embeddable-views>

**Comment by hythloda** [comment]
We have tried several iterations of this in the WP site and could not get it to look good on both mobile desktop with the current iterations. We can explore more later with time but do not have the bandwidth at the moment. Some of our testing is below

The code provided gives just one looks like this on desktop
<img width="1197" alt="Screenshot 2024-03-20 at 4 06 43 PM" src="https://github.com/ossf/ossf-landscape/assets/51727488/7292b82f-37e1-4c67-8b67-dd040e3c7554">

meanwhile on mobile
![IMG_6885](https://github.com/ossf/ossf-landscape/assets/51727488/dc7db1c6-c9c4-4380-807b-bcbafa197bff)

The only way I saw to fix this was setting the size of each element and that was functional on Desktop

<img width="671" alt="Screenshot 2024-03-20 at 4 14 46 PM" src="https://github.com/ossf/ossf-landscape/assets/51727488/3847e77d-2fb6-49d9-b32b-e80d86e2ea1b">

However mobile did not show anything still.

**Comment by tegioz** [comment]
Hi @hythloda 👋 

Did you enable `iframe-resizer`? It works slightly different than in the previous landscape (please see the demo referenced below).

> [!NOTE]
> In addition to the customization options available in the embed setup view, it's also possible to embed views using [iframe-resizer](https://github.com/davidjbradshaw/iframe-resizer). This feature can be enabled by adding `iframe-resizer=true` to the embed url ([demo](https://codepen.io/cynthiasg/pen/WNmQjje)).

Is the source code of that website publicly available? If you still run into issues after trying the suggestion above, we'd be happy to take a look at it and submit a PR if needed 🙂

**Comment by hythloda** [comment]
We did try the iframe-resizer and it still had a scroller
```
  <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js"></script>  
  <iframe id="iframe-landscape" src="https://openssf.landscape2.io/embed/embed.html?key=openssf-members--premier&headers=true&category-header=true&category-in-subcategory=false&item-name=false&title-uppercase=false&title-alignment=left&title-font-family=sans-serif&title-font-size=13&style=clean&size=xl&items-alignment=left&bg-color=%23331f6b&fg-color=%23ffffff&iframe-resizer=true" style="width: 1px; min-width: 100%; min-height: 100px; border: 0;"></iframe>
  <script>
    iFrameResize({ }, '#iframe-landscape');
  </script>
```

**Comment by cynthia-sg** [comment]
Hi @hythloda  👋

> We did try the iframe-resizer and it still had a scroller
> 
> ```
>   <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js"></script>  
>   <iframe id="iframe-landscape" src="https://openssf.landscape2.io/embed/embed.html?key=openssf-members--premier&headers=true&category-header=true&category-in-subcategory=false&item-name=false&title-uppercase=false&title-alignment=left&title-font-family=sans-serif&title-font-size=13&style=clean&size=xl&items-alignment=left&bg-color=%23331f6b&fg-color=%23ffffff&iframe-resizer=true" style="width: 1px; min-width: 100%; min-height: 100px; border: 0;"></iframe>
>   <script>
>     iFrameResize({ }, '#iframe-landscape');
>   </script>
> ```


The code you shared seems to be working fine -no scroll-, you can see it in action in this codepen: https://codepen.io/cynthiasg/pen/QWPOdYm (I applied some minor changes to the styles to make it look similar to your current site, but it's mostly the same).

I'd be happy to assist further if you could point me to the site's source code. Please note that if you have multiple iframes you need to call iFrameResize(...) for each using the corresponding id. You can also see how this is being used in the [Kubernetes](https://kubernetes.io/partners/) or [CNCF](https://www.cncf.io/about/members/)'s website, just in case it helps.

**Comment by hythloda** [comment]
Must be a wordpress plug issue then here is what I see after uploading it
[screen-capture.webm](https://github.com/ossf/ossf-landscape/assets/51727488/24f3c304-8f55-4742-abca-b5a1861f0b36)

**Comment by hythloda** [comment]
I have tried various ways of getting it to work with the rawJS block and it always looks the same with the small scrolling window. We have plans on potentially migrating to a new site and landscape1 works fine until such a time.
Attempt:
<img width="1054" alt="3" src="https://github.com/ossf/ossf-landscape/assets/51727488/c4c128f5-c8c6-4227-a3bf-a0ace25b9fc6">
Attempt:
<img width="1049" alt="2" src="https://github.com/ossf/ossf-landscape/assets/51727488/e7a7f4b5-4e12-4e8f-a3e6-d75b41c4e14f">
Attempt:
<img width="1062" alt="1" src="https://github.com/ossf/ossf-landscape/assets/51727488/c6fb07cb-1128-41db-9171-e7ee4465cd35">

**Comment by jmertic** [comment]
This appears to be all fixed, and we are fully over to landscape2 - closing.

https://openssf.org/about/members/
